src/test/java/org/hiero/sketch/MembershipTest.java

```diff
@@ -0,0 +1,79 @@
+package org.hiero.sketch;
+
+import org.hiero.sketch.table.FullMembership;
+import org.hiero.sketch.table.PartialMembershipDense;
+import org.hiero.sketch.table.PartialMembershipSparse;
+import org.hiero.sketch.table.api.IRowIterator;
+import org.junit.Test;
+import org.omg.PortableInterceptor.SYSTEM_EXCEPTION;
+
+/**
+ * Created by uwieder on 10/24/16.
+ */
+public class MembershipTest {
+    private int size = 10;
+    @Test
+    public void TestFullMembership(){
+        System.out.println("Testing FullMemebership");
+        FullMembership FM = new FullMembership(size);
+        IRowIterator IT = FM.getIterator();
+        int tmp = IT.getNextRow();
+        while (tmp >=0){
+            System.out.print(tmp+",");
```

Comment:
Don't print in tests - we'll have hundreds of them and we won't ever read this output again.